posthog/models/entity/util.py

```diff
@@ -7,29 +7,44 @@
 
 
 def get_entity_filtering_params(
```

Comment:
Is the behavioral change here an unrelated fix?

Reply:
Do you mean a behavioral change other than support for multiple entities (logically OR'd)? This is primarily for formula mode Trends insights, where multiple entities become relevant for a single data point, which is where we need to show the properties timeline for _all_ of those entities.